.github/workflows/ci-pr-validation.yaml

```
@@ -219,13 +219,36 @@ jobs:
       - name: Build packages
         run: pkg/${{matrix.pkg.type}}/docker-build-${{matrix.pkg.type}}-${{matrix.cpu.platform}}.sh build:latest
 
+  cpp-build-macos:
+    timeout-minutes: 120
+    name: Build CPP Client on macOS
+    runs-on: macos-12
+    needs: unit-tests
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: brew install openssl protobuf boost boost-python3 googletest zstd snappy
```

```suggestion
        run: brew install openssl protobuf boost zstd snappy
```

Remove unused dependencies